src/desktop/apps/artwork/components/commercial/templates/acquire.jade

```diff
@@ -5,4 +5,4 @@ if artwork.is_acquireable && (newBuyNowFlow || auctionPartner)
     input( type='hidden', name='artwork_id', value= artwork.id )
 
     button.avant-garde-button-black.is-block( class='js-artwork-acquire-button' )
-      | Buy
+      | Buy Now
```

Comment:
I _think_ this should be `Buy now` since we're moving towards this kind of casing

Reply:
I just followed what we have for the auctions work already, but I guess should update all. It does not really matter because in the current button style - text is all caps:)